src/main/java/com/gdschongik/gdsc/domain/study/domain/AssignmentHistoryGrader.java

```diff
@@ -0,0 +1,52 @@
+package com.gdschongik.gdsc.domain.study.domain;
+
+import static com.gdschongik.gdsc.domain.study.domain.SubmissionFailureType.*;
+import static com.gdschongik.gdsc.global.exception.ErrorCode.*;
+
+import com.gdschongik.gdsc.global.annotation.DomainService;
+import com.gdschongik.gdsc.global.exception.CustomException;
+import com.gdschongik.gdsc.global.exception.ErrorCode;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@DomainService
+public class AssignmentHistoryGrader {
+
+    public static final int MINIMUM_ASSIGNMENT_CONTENT_LENGTH = 300;
```

Comment:
```suggestion
    private static final int MINIMUM_ASSIGNMENT_CONTENT_LENGTH = 300;
```
클래스 내에서만 사용되는 것 같은데 public이어야 할 필요가 있을까요?